codalab/lib/codalab_manager.py

```diff
@@ -605,6 +612,25 @@ def set_current_worksheet_uuid(self, address, worksheet_uuid):
             if 'worksheet_uuid' in session: del session['worksheet_uuid']
         self.save_state()
 
+    def check_version(self, server_version):
+        # Enforce checking version at most once every 24 hours
+        now = datetime.datetime.now()
+        last_check_dt = formatting.parse_datetime(self.state.get('last_check_version', formatting.datetime_str(datetime.datetime.utcfromtimestamp(0))))
```

Comment:
Replace last_check_version with something like last_check_version_datetime for clarity